### ISSUE TITLE
Fail precondition check for sfc_alb_dir_vis with ne120

### ISSUE THREAD

**Issue by ndkeen** [E3SM-Project/scream#1575]
With April 27th master, tried a ne120 cime case and see this error after 1st step on PM:

```
  1: terminate called after throwing an instance of 'std::logic_error'
  1:   what():  /global/cfs/cdirs/e3sm/ndk/se02-apr27/components/scream/src/share/atm_process/atmosphere_process.cpp:244: FAIL:
  1: false
  1: Error! Failed pre-condition check (cannot be repaired).
  1:   - Atm process name: Radiation
  1:   - Property check name: sfc_alb_dir_vis within interval [0, 1]
  1:   - Atmosphere process MPI Rank: 1
  1:   - Message: FieldWithinIntervalCheck failed; min = 0.700549; max = 1.254542
```

Will run again to try getting snapshot after 1st step.

**Comment by PeterCaldwell** [comment]
Ok, it's great that the condition checks spit out the max and min values now. I was hoping to see that we were just roundoff-level outside of bounds but an albedo of 1.25 is far from roundoff. Yeah, it would be useful to see the output after step 1: I'm curious whether this excessive value is a single location or widespread.

@brhillman - do you know if albedo is taken directly from the surface schemes (i.e. is this not our fault)?

Also, do we know if EAM was employing limiters which just forced albedo into the acceptable range?

**Comment by ndkeen** [comment]
Currently unable to write snapshots for ne120 (ne30 ok) due to 
```
0: PIO: FATAL ERROR: Aborting... FATAL ERROR: NetCDF: One or more variable 
  sizes violate format constraints 
  (file = SCREAMv1_output_hydrostatic.INSTANT.Steps_x1.np128.0001-01-01.000230.nc) (/global/cfs/cdirs/e3sm/ndk/se02-apr27/externals/scorpio/src/clib/pioc_support.c: 3921)
```
which I thought was something we already fixed

**Comment by brhillman** [comment]
We do clip in EAM, but the tolerance is set to `0.01`:
```
      call handle_error(clip_values( &
         albedo_dir, 0._r8, 1._r8, trim(subname) // ': albedo_dir', tolerance=0.01_r8) &
      )
      call handle_error(clip_values( &
         albedo_dif, 0._r8, 1._r8, trim(subname) // ': albedo_dif', tolerance=0.01_r8) &
      )
```
This is also after we map the broadband albedos to individual RRTMGP bands. I'll do a quick test and see what the max values are, but 1.25 seems too large. I do think these more or less get read in directly from the surface models, the precondition checks should be checking the values that have been copied into the FM from the surface models I think, before any manipulation is done on them. Note these are also "input" fields into `atmosphere_radiation`, so are not modifiable there.

**Comment by AaronDonahue** [comment]
@ndkeen , can you try writing the coupler history output instead of the SCREAM output for now?  I would be curios if the coupler is passing albedo values greater than 1.0.  I think this is accomplished by changing the `HIST_N` and `HIST_OPTION` variables in the case directory.  So maybe

```
./xmlchange HIST_N=1
./xmlchange HIST_OPTION=nsteps
```

Then we can look at the max/min of the variables `x2a_Sx_avsdf` and `x2a_Sx_anidf`

**Comment by PeterCaldwell** [comment]
I'd also be curious whether HIST_N and HIST_OPTION have any effect... lately I've been trying to print these cpl.hi files but never get any.

**Comment by PeterCaldwell** [comment]
> FATAL ERROR: Aborting... FATAL ERROR: NetCDF: One or more variable sizes violate format constraints

Is the reason for this that we're writing netcdf classic still? I thought that switching output formats was the solution to this problem last time?

**Comment by AaronDonahue** [comment]
> I'd also be curious whether HIST_N and HIST_OPTION have any effect... lately I've been trying to print these cpl.hi files but never get any.

I just did this for my ne30 run on syrah and am getting a cpl hist every timestep.  Not sure if `env_test` or `env_run` needs the change.  I decided to change both.

**Comment by PeterCaldwell** [comment]
Ah, that probably explains it. I didn't specify a file and I bet it was just changing env_run but using env_test.

**Comment by ndkeen** [comment]
I'm just having trouble remembering the details of the previous issues, but I think we fixed things by honoring all of the PIO related settings in CIME which would use pnetcdf (and not saying that's the issue again here). 

I'm trying suggestion of trying to write a coupler history file -- been waiting in Q

**Comment by bartgol** [comment]
Yes, we _should_ be using the correct NC format now.

**Comment by ndkeen** [comment]
I was able to write cpl history file. However, PM is down for maint now. This does indicate we can probably write certain data files OK.

**Comment by brhillman** [comment]
I tried a v0 run with ne120, and now I’m getting a similar error there, while I wasn’t before rebasing the initial condition generation branch. The values don’t seem as bad, around 1.02, but this seems to be larger than they were before and now fails our tolerance check in v0.

**Comment by brhillman** [comment]
Also seeing very hot temperatures in the v0 cases with updated repo…up to 390K. So something is going off the rails with this config, even with v0. @ndkeen what was the full resolution you are running? These cases were with `ne120_r0125_oRRS18to6v3`.

**Comment by PeterCaldwell** [comment]
Ugh. What machine are you running on, Ben. Cori? How many steps can you run before you crash? When was the previous hash that "worked" from? Should we bisect? I don't think we've intentionally changed anything in v0 in ages.

**Comment by brhillman** [comment]
I'm on cori, yes. This was failing on first step. I'm still trying to reproduce Noel's documented sfc_alb_dir_vis error with v1 though, with recent master it's crashing in some other way with no error reporting.

**Comment by PeterCaldwell** [comment]
It seems like making sure v0 works might be the best first step since we haven't changed anything in v0 for a long time/didn't expect it to have problems?

**Comment by brhillman** [comment]
Yeah, the odd thing is that the values did not look as extreme with v0 as they did with v1, but I'd expected them to be identical since those should be coming from the surface models. This was also just with ne120, I haven't tried ne4 or ne30. I suspect ne4 is fine since we test that. It also might depend on what grids the surface models are on; my first thought was that maybe something about horizontal remapping between surface models and atm was introducing extrema outside the original bounds.

**Comment by bartgol** [comment]
The obvious suspect is the upstream merge, at this point.
